doc/releases/release-notes-3.6.rst

```diff
@@ -145,6 +145,9 @@ Drivers and Sensors
 
 * I3C
 
+  * The Legacy Virtual Register defines have been renamed from I3C_DCR_I2C_* to
+    I3C_LVR_I2C_*.
```

Comment:
You need to enclose the names in double backticks.

Reply:
done